### PR TITLE
Viewer IIIF Drag-N-Drop

### LINKED ISSUE #4106: Add "IIIF" icon with a link to the IIIF manifest to UV
Add "IIIF" icon with a link to the IIIF manifest in the lower bar of the viewer, next to the share icon.

Example of similar UI:

https://artmuseum.princeton.edu/collections/objects/36947

## Technical Implementation Notes

We'll have to add this to the `uv_manager` JS. We can probably just create the button and inject it into the viewer that's rendered on the page.

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #4106

### CODE REVIEW

**Comment by tpendragon** [comment]
Screen Shot 2020-07-20 at 10 38 24 AM

File: app/javascript/viewer/uv_manager.js [hunk 1]
```diff
@@ -36,6 +37,31 @@ export default class UVManager {
     this.cdlTimer.initializeTimer()
   }
 
+  addIIIFIcon () {
+    const existingButton = document.querySelector('a.iiif-drag')
+    if (existingButton !== null) {
+      return
+    }
+    const shareButton = document.querySelector('button.share')
+    const link = document.querySelector('a.imageBtn.iiif').href
```
Comment (hackmastera) [inline app/javascript/viewer/uv_manager.js:46]: I didn't know that was there! A comment describing this location would be helpful, something like `// pull the link from the iiif button inside the share popup`

**Comment by tpendragon** [comment]
@hackmastera Drag-n-drop works in UV, I think the problem on staging is a CORS config issue for the staging nginx config. I added the button to the mobile view too.

**Comment by tpendragon** [comment]
Dragging from a complete open resource seems to work: https://figgy-staging.princeton.edu/catalog/8ddd39a5-b853-48c7-b9cd-35296e5e09f7

**Pull request merged**